packages/fxa-settings/src/components/Settings/FlowRecoveryKeyInfo/index.tsx

```diff
@@ -0,0 +1,110 @@
+export const backwardNavigationEventName = 'navigate-forward';
+export const forwardNavigationEventName = 'navigate-backward';
+
+export const FlowRecoveryKeyInfo = ({
+  navigateForward,
+  navigateBackward,
+}: FlowRecoveryKeyInfoProps) => {
+  usePageViewEvent(viewName);
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedContainerPageTitle = ftlMsgResolver.getMsg(
+    'flow-recovery-key-info-page-title',
+    'Account Recovery Key'
+  );
+
+  return (
+    <FlowContainer
+      title={localizedContainerPageTitle}
+      customBackButtonTitle="Settings"
+      customBackButtonLocalizationId="flow-recovery-key-info-back-button-title"
+      onBackButtonClick={() => {
+        logViewEvent(viewName, backwardNavigationEventName);
+        navigateBackward();
+      }}
+    >
+      <div className="w-full flex flex-col">
+        <ProgressBar currentStep={1} numberOfSteps={4} />
+        <SecurityShieldImage className="mx-auto my-6" />
+        <FtlMsg
+          id="flow-recovery-key-info-header"
+          elems={{
+            heading: (
+              <h2 className="font-bold text-xl mb-4">
+                Create an account recovery key in case you forget your password
+              </h2>
+            ),
+          }}
+        >
+          <h2 className="font-bold text-xl mb-4">
+            Create an account recovery key in case you forget your password
+          </h2>
+        </FtlMsg>
+        <ul>
+          <ShieldIconListItem>
+            <FtlMsg id="flow-recovery-key-info-shield-bullet-point">
+              <p className="text-sm">
+                We encrypt browsing data –– passwords, bookmarks, and more. It’s
+                great for privacy, but it means we can’t recover your data if
+                you forget your password.
+              </p>
+            </FtlMsg>
+          </ShieldIconListItem>
+          <KeyIconListItem>
+            <FtlMsg id="flow-recovery-key-info--key-bullet-point">
+              <p className="text-sm">
+                That’s why creating an account recovery key is so important ––
+                you can use your key to get your data back
+              </p>
+            </FtlMsg>
+          </KeyIconListItem>
+        </ul>
+        <FtlMsg
+          id="recovery-key-add-wizard-flow-first-step-cta-text"
+          elems={{
+            button: (
+              <button
+                className="cta-primary cta-base-p text-sm my-2"
+                type="button"
+                onClick={() => {
+                  logViewEvent(viewName, forwardNavigationEventName);
+                  navigateForward();
+                }}
+              >
+                Start creating your recovery key
+              </button>
+            ),
+          }}
+        >
+          <button
+            className="cta-primary cta-base-p text-sm my-2"
+            type="button"
+            onClick={() => {
+              logViewEvent(viewName, forwardNavigationEventName);
```

Comment:
Should logging happen on button click or on flow page itself?

Reply:
Oh I must be grabbing the wrong metrics method -- I meant to use the one we use for actions. I'll look into this

Reply:
Okay I checked and this is the method they're calling within action callbacks on the existing page. I think that matches roughly with what we're doing here (just logging that the user is navigating forward) but I see what you mean -- maybe it's redundant since we'll also log when they hit the next step in the flow? What do you think?